spotbugs/src/main/java/edu/umd/cs/findbugs/detect/DumbMethods.java

```diff
@@ -490,14 +490,16 @@ public void initMethod(Method method) {
 
         @Override
         public void sawOpcode(int seen) {
-            if (seen == Const.INVOKEVIRTUAL && "java/util/Random".equals(getClassConstantOperand())
+            if (seen == Const.INVOKEVIRTUAL && ("java/util/Random".equals(getClassConstantOperand()) || "java/security/SecureRandom".equals(
```

Comment:
Perhaps `getClassConstantOperand()` should be extracted to a local variable.

Reply:
Done. Thanks for your good suggestion :)